tests/bsim/bluetooth/audio/src/csip_set_member_test.c

```diff
@@ -22,6 +24,14 @@ struct bt_csip_set_member_register_param param = {
 		      0x22, 0xfd, 0xa1, 0x21, 0x09, 0x7d, 0x7d, 0x45 },
 };
 
+struct bt_csip_set_member_register_param param_unranked = {
+	.set_size = 3,
+	.lockable = true,
+	/* Using the CSIS test sample SIRK */
+	.set_sirk = { 0xcd, 0xcc, 0x72, 0xdd, 0x86, 0x8c, 0xcd, 0xce,
+		      0x22, 0xfd, 0xa1, 0x21, 0x09, 0x7d, 0x7d, 0x45 },
+};
```

Comment:
Any reason to have a duplicate param, rather than updating the `rank` value of the existing one?